### PR TITLE
fixed format, added pic, changed text

### PR DESCRIPTION
**Pull request by cwlainson**

### CODE REVIEW

**Comment by cwlainson** [comment]
need to resubmitt

**Review by gc824 — CHANGES_REQUESTED** [review]
All of this have to go into CSS file

- h1 style="background-color: green">Our Story</h1
- p style="background-color: tomato">Img
this has to go in img folder

This in img folder
- src="/Users/christopherlainson/Desktop/workspace/PizzaParty/img/party_boys.jpg"

iframe styling has to go in CCS file:

- width="560"
- height="315"
- frameborder="0"

File: ourstory.html [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
-        <img src="/Users/christopherlainson/Desktop/workspace/PizzaParty/img/party_boys.jpg" 
-        Alt="Pardi Bros Havana NYE 1958 colorized" style="width:560px;height:325px"><p>Carlo and Giovanni Pardi NYE 
-            Havana, Cuba 1958 (colorized).</p>
-        <iframe width="560" height="315" src="https://www.youtube.com/embed/CPEwAA6G42I?start=15" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-
-</body>
+<html>
+  <body>
+    <head>
+      <h1 style="background-color: green">Our Story</h1>
```
Comment (cwlainson) [inline ourstory.html:5]: move to css stlye sheet

File: ourstory.html [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
+      <p style="background-color: tomato">
+        The Pizza Party story is really the story about America, from Naples to
+        New York Cityto Buffalo, NY. In 1938 at the age of 12 and 15 Giovanni and Carlo
```
Comment (cwlainson) [inline ourstory.html:8]: space between "city to"

**Comment by billythakidd04** [comment]
This is being redone on an updated branch

**Pull request closed without merging**